src/llm/setup.py

```diff
@@ -0,0 +1,21 @@
+from setuptools import setup, find_packages
+
+install_requires = open("requirements.txt").read().strip().split("\n")
+
+setup(
+    name="aqueduct-llm",
+    version="0.2.11",
```

Comment:
Do we want to match this number with the version of aqueduct-ml or keep it as a diff numbering system?

Reply:
Let's version this as `0.0.1` for now.